app/renderer/js/utils/default-util.ts

```diff
@@ -1,6 +1,6 @@
-const fs = require('fs');
+import fs = require('fs');
 
-let app = null;
+let app: Electron.App;
```

Comment:
How does this work? Two doubts here -
* Should it be `null`?
* We haven't imported electron here?

Reply:
The `let app: Electron.App` tells typescript that it will be an electron app instance when initialized so this is correct. If assigned some other value to it later on it will complain.

Reply:
We should use `let app: Electron.App = null;` here?

Reply:
Done! I changed this one instance to `let app: Electron.App = null;`. Also, I realized I missed answering the second bullet, `Electron` comes from the electron typescript declaration file which means that it is automatically know by typescript without importing it.